addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1037,6 +1041,10 @@ def _channel_seen(self, last_message_id=None):
         if last_message_id is not False and not last_message:
             return
         self._set_last_seen_message(last_message)
+        current_member = self.env['discuss.channel.member'].search(
```

Comment:
miss-match of quotes (I'd suggest double, didn't we apply black at some point on that file?)